.env.dev

```
@@ -1,2 +1,2 @@
 REACT_APP_TEXT="I'm REACT_APP_TEXT from .env"
-VITE_BASE_URL = "http://localhost:3000"
+VITE_BASE_URL = "https://sampoom.store"
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)